### PR TITLE
allow classes + keys in anchors

### PR DESCRIPTION
**Pull request by hxpaul**
#### What does this PR do? (please provide any background)

Allows `<Anchor />` to be overridden with an array of classes so we can replace the Anchor used in tripapp with this one. Also adds an optional `key` for the same reason. Also updates `make-up`, hey our SNYK report is good.

#### What tests does this PR have?

None

#### How can this be tested?

Read the code, run the tests. Nothing is using this feature yet but I'll update tripapp to use this in another pr.

---

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [ ] :+1:

**Review 2** \*
- [ ] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

File: docs/public/assets/bundle.js [hunk 1]
```diff
@@ -40,6 +40,13 @@ var _extends = Object.assign || function (target) { for (var i = 1; i < argument
 
 var React = require('react');
 var DataAttributesMixin = require('react-data-attributes-mixin');
+var _React$PropTypes = React.PropTypes;
```
Comment (hxpaul) [inline docs/public/assets/bundle.js]: this file is generated of course, don't review this

**Comment by hx-markterry** [comment]
taken

**Comment by jodiedoubleday** [comment]
I still have an issue with classes being added to that component. I would prefer if the styling that was needed on that anchor be added to the toolkit. Rather than adding a class in order to write their own styles.

`key` :+1:   `class` :-1:

**Comment by hxpaul** [comment]
I think this was the right solution and I was depending on this change to update the react version, but I'll close this one and have a rethink. Not a priority for now though.

**Pull request closed without merging**